cms/utils/compat/forms.py

```diff
@@ -1,14 +1,13 @@
 # -*- coding: utf-8 -*-
+from django.conf import settings
 from django.utils import importlib
 from django.db import models
 
-from .dj import is_user_swapped, user_model_label
-
 
 # overide with custom classes if they exist
-if is_user_swapped:  # pragma: no cover
+if settings.AUTH_USER_MODEL != 'auth.User':  # pragma: no cover
```

Comment:
I don't mind changing it to `if User._meta.swapped` if you prefer that. Mostly chose this to avoid an import.